hoptimator-operator/src/main/java/com/linkedin/hoptimator/operator/HoptimatorOperatorApp2.java

```diff
@@ -0,0 +1,41 @@
+package com.linkedin.hoptimator.operator;
+
+import io.kubernetes.client.informer.SharedInformerFactory;
+import io.kubernetes.client.extended.controller.Controller;
+import io.kubernetes.client.extended.controller.ControllerManager;
+
+import com.linkedin.hoptimator.k8s.K8sContext;
+import com.linkedin.hoptimator.k8s.K8sApiEndpoints;
+import com.linkedin.hoptimator.operator.pipeline.PipelineReconciler;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.time.Duration;
+
+public class HoptimatorOperatorApp2 {
```

Comment:
Can we name it something like `HoptimatorOperatorPipelineApp` if this will eventually become the default?

Reply:
That's interesting. I was expecting to need additional controllers (via ControllerProvider SPI) but the new planner makes controllers MOSTLY irrelevant. We still need operators for Flink etc, but they don't necessarily need to be bundled here. So maybe it's fine to have a PipelineOperatorApp that just runs the pipeline controller? I think that makes sense.

Reply:
I'll leave it up to you, don't love the `2` at the end of it but I'm not opposed to keeping it this way if we want to bundle the other operators to make everything cleaner

Reply:
Let's make this just `PipelineOperatorApp`.